on-boot-script/dpkg-build-files/debian/postinst

```diff
@@ -23,8 +23,8 @@ case "$1" in
        scp -P "$(cat /etc/unifi-os/ssh_proxy_port)" -o StrictHostKeyChecking=no -q /usr/share/udm-boot/on_boot.sh root@localhost:/mnt/data/on_boot.sh
        /sbin/ssh-proxy 'chmod +x /mnt/data/on_boot.sh && mkdir -p /mnt/data/on_boot.d'
 
-       deb-systemd-invoke enable udm-boot
-       deb-systemd-invoke start udm-boot
+       systemctl enable udm-boot
```

Comment:
systemd supports the `--now` flag since version 220. As far as I can tell, unifi-os uses version 241.

`systemctl enable --now udm-boot` seems to work in my case.

Reply:
good point, thanks